### PR TITLE
Typescript: Migrate @storybook/mithril

### PR DESCRIPTION
**Pull request by hcz**
Issue: #5030

## What I did
Migrate @storybook/mithril to Typescript

## How to test

Everything should function the same, except the package should now have type definitions.

### CODE REVIEW

**Comment by hcz** [comment]
@shilman What should I do more? Some checks were failed

**Comment by shilman** [comment]
@hcz Looks good -- I just added an extra label! I think one of our typescript posse should review & then we can merge.

**Review by ndelangen — CHANGES_REQUESTED** [review]

File: app/mithril/package.json [hunk 1]
```diff
@@ -31,14 +32,17 @@
...
     "@babel/plugin-transform-react-jsx": "^7.3.0",
+    "@storybook/addons": "5.3.0-alpha.15",
     "@storybook/core": "5.3.0-alpha.15",
     "core-js": "^3.0.1",
     "global": "^4.3.2",
     "regenerator-runtime": "^0.13.3",
     "ts-dedent": "^1.1.0"
   },
   "devDependencies": {
+    "@types/mithril": "^2.0.0",
```
[the remark this answers is not in the document]
Comment (hcz) [reply app/mithril/package.json]: > should be a regular dependency IMHO

Ok, fixed c925c87

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
good to merge!

**Comment by ndelangen** [comment]
I invited you to the org @hcz !

**Review by shilman — APPROVED** [review]
Great job @hcz !!! 🚀

**Comment by davidroman0O** [comment]
I know it's merged but anyone have an example of Mithril + TypeScript ?

Or this merge is not yet release? maybe?

I tried to setup one (multiple times) and I still have a lot of errors with `esModuleInterop` flag and others from simple TypeScript helloworld like components.

**Comment by ndelangen** [comment]
@davidroman0O This feature will be released in 5.3 (see milestone)
You can try this today by upgrading to the beta.

We'd love to hear your feedback. A more detailed report of what you tried, and what errors you got would help./

**Comment by davidroman0O** [comment]
@ndelangen  Oh yeah ! 😄 

Thx!

**Pull request merged**